README.md

```diff
@@ -1,38 +1,135 @@
-This is a [Next.js](https://nextjs.org/) project bootstrapped with [`create-next-app`](https://github.com/vercel/next.js/tree/canary/packages/create-next-app).
+# Polis
 
-## Getting Started
+---
 
-First, run the development server:
+Polis is a platform dedicated to showcasing web3 tools, templates, and boilerplates. Frequently, these resources remain isolated or challenging to find within the web3 landscape. Polis is committed to addressing this challenge by offering a solution to enhance their discoverability and accessibility.
 
-```bash
+---
+
+## How It Works
+
+![polis](./public/polis.png)
+
+**If you are a Contributor:**
+
+- You can submit new tools/boilerplates/templates
+- You will also be able to edit the application later
+
+Note: Contributors will need to connect their wallet to be able to submit and edit the applications.
+
+**If you are a User:**
+
+- You can explore tools, templates and boilerplates on Polis
+- You will find all necessary web3 tools/templates/boilerplates in one place
+
+**What are the main features?**
+
+- Connect Wallet
+- Submit New Applications: _Can contribute and share your own application on the platform_
+- Edit Applications: _Contributors are able to edit the applications they've submitted_
+- Editor's Pick: _Users can discover the applications most used by the community_
+
+## Technical Details:
+
+| Name   | Link                            | Text           |
+| ------ | ------------------------------- | -------------- |
+| Nodejs | https://nodejs.org/en/download/ | Latest Version |
+| Infura | https://infura.io/              |                |
+
+### **Libraries used**
+
+Here's an overview of the included frameworks and tools.
+
+- **Next.js** - A frameworks fo React applications that is focused on server-rendered and minimalistic approach.
```

Comment:
let's add more to nextjs part. mentioning that we use nextjs's latest `app` route and server actions. 
since server actions (https://nextjs.org/docs/app/building-your-application/data-fetching/server-actions)  still in alpha, which means it is unstable